modules/fedimint-mint-client/src/lib.rs

```diff
@@ -74,14 +74,27 @@ const MINT_BACKUP_RESTORE_OPERATION_ID: OperationId = OperationId([0x01; 32]);
 
 pub const LOG_TARGET: &str = "client::module::mint";
 
+#[derive(Clone, Debug, Decodable, Encodable, Deserialize, Serialize)]
```

Comment:
The `Serialize`/`Deserialize` impl does not what we want here, the struct should be encoded as one string if the encoding format is human readable. If you aren't familiar with custom implementations of `Serialize`/`Deserialize` you could just keep the `serde_ecash` module around for now.

Reply:
Ah I think I follow. We'd still like for the serialization of the notes (`OOBNotes`) to be one string containing a base-64 byte encoding of the struct. I will write a custom serializer.

Reply:
There's a `serde_as_encodable_hex` for exactly this already.